src/Game.cpp

```diff
@@ -390,6 +391,8 @@ bool Game::Run()
 	auto& textureManager = resources.GetTextures();
 	auto& animationManager = resources.GetAnimations();
 
+	const auto citadelOutsideMeshesPath = _fileSystem->FindPath(_fileSystem->CitadelPath() / "OutsideMeshes");
+	meshManager.Load("temple", "temple", citadelOutsideMeshesPath / "b_first_temple_l3d.zzz");
```

Comment:
You should give it the name of the file `b_first_temple_l3d`

Reply:
I would like to see all `*.zzz` files in `OutsideMeshes/` loaded here using the stem as the name.
See #378 with `*.raw` files